modules/gltf/src/lib/parsers/parse-gltf.ts

```diff
@@ -208,7 +209,22 @@ async function loadImage(
   assert(arrayBuffer, 'glTF image has no data');
 
   // Call `parse`
-  const parsedImage = await parse(arrayBuffer, ImageLoader, {}, context);
+  let parsedImage = await parse(
+    arrayBuffer,
+    [ImageLoader, BasisLoader],
+    {mimeType: image.mimeType, basis: {format: selectSupportedBasisFormat()}},
+    context
+  );
+
+  if (parsedImage && parsedImage[0]) {
```

Comment:
- I don't like that we have to do this check. It would be good if both loaders returned a single image, with the same format. The imageloader does not return an array.  Maybe BasisLoader can have an option to only return one image?

- Also, can `parsedImage` ever be null? Wouldn't `parse` throw an exception? If it doesn't can we fix that?

Reply:
It filters `ImageBitmap`